module/VuFind/tests/integration-tests/src/VuFindTest/Mink/AccountActionsTest.php

```diff
@@ -329,8 +330,8 @@ public function testDefaultPickUpLocation(): void
         $this->submitCatalogLoginForm($page, 'catuser', 'catpass');
 
         // Check the default library and possible values:
-        $userTable = $this->getTable(User::class);
-        $this->assertSame('', $userTable->getByUsername('username2')->getHomeLibrary());
+        $userTable = $this->getDbService(UserService::class);
```

Comment:
For consistency of naming, it's probably worth replacing $userTable with $userService in this file.